src/types/user_operation.rs

```diff
@@ -155,6 +156,100 @@ pub struct UserOperationReceipt {
     pub receipt: TransactionReceipt,
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct UserOperationPartial {
```

Comment:
@zsluedem Do you know of any better approach?

Reply:
No, I don't.